api/controlplane/kubeadm/v1beta2/kubeadm_control_plane_types.go

```diff
@@ -824,7 +824,7 @@ type LastRemediationStatus struct {
 // +kubebuilder:subresource:status
 // +kubebuilder:subresource:scale:specpath=.spec.replicas,statuspath=.status.replicas,selectorpath=.status.selector
 // +kubebuilder:printcolumn:name="Cluster",type="string",JSONPath=".metadata.labels['cluster\\.x-k8s\\.io/cluster-name']",description="Cluster"
-// +kubebuilder:printcolumn:name="Available",type="string",JSONPath=`.status.conditions[?(@.type=="Available")].status`,description="Cluster pass all availability checks"
+// +kubebuilder:printcolumn:name="Cluster Available",type="string",JSONPath=`.status.conditions[?(@.type=="Available")].status`,description="Cluster pass all availability checks"
```

Comment:
Isn't this description wrong instead of the column name?
Isn't this showing the available condition of KCP, not the Cluster?

Reply:
Ah, yeah, it's correct. It's not condition for Cluster. Thanks. I'd revise the this description and all other descriptions.